boards/nxp/mimxrt1170_evk/board.cmake

```diff
@@ -6,10 +6,7 @@
 
 if(CONFIG_SOC_MIMXRT1176_CM7 OR CONFIG_SECOND_CORE_MCUX)
  board_runner_args(pyocd "--target=mimxrt1170_cm7")
- board_runner_args(jlink "--device=MIMXRT1176xxxA_M7" "--reset-after-load")
```

Comment:
don't we want to reset after load for the more expected use case of actually booting with the ROM? or is this just what is now the default?

Reply:
This should be the default, this seems to be the default for the other boards as well which is one step that prevented them from running into this issue.